rubygems-org-api.md

```diff
@@ -142,7 +142,7 @@ response is received.
 List all gems that you own. Returns an array of the JSON or YAML representation
 of gems you own.
 
-    $ curl -H 'Authorization:701243f217cdf23b1370c7b66b65ca97' \
+    $ curl -H 'Authorization:rubygems_701243f217cdf23b1370c7b66b65ca97' \
```

Comment:
if you want to be really accurate, the hash length is now of 48 chars. https://github.com/rubygems/rubygems.org/blob/master/app/controllers/concerns/api_keyable.rb#L18